.github/workflows/mac-build.yml

```diff
@@ -0,0 +1,140 @@
+name: ODBC Driver for Mac
+
+on:
+  push:
+    paths:
+      - '.github/workflows/win-build.yml'
```

Comment:
I assume we want `mac-build.yml` here?

Reply:
Yes, good catch here. I have created [Jira](https://bitquill.atlassian.net/browse/GAC-60) to fix this.